src/main/java/net/sourceforge/tess4j/util/ImageIOHelper.java

```diff
@@ -105,49 +105,48 @@
 
         ImageReader reader = readers.next();
 
```

Comment:
@nguyenq the changes of this file are safe and makes sense for me, he did da great error handling job and added nice null checks, though crappy format.